### PR TITLE
DB migration script (+doc): csub collection

### PR DESCRIPTION
**Pull request by fgalan**
Part of #1851 

It has been tested with real data coming from orion.lab. In addition, all the skip conditions has been manually tested with "hacked" documents at csub.

### CODE REVIEW

File: doc/manuals/admin/upgrading_crossing_1-3-0.md [hunk 1]
```diff
@@ -30,11 +31,29 @@ at least one entity document.
 
         python mdsvector2mdsobject.py orion
 
--   If you get the following message, there is some problem that needs
+-   If you get some of the following messages, there is some problem that needs
```
Comment (kzangeli) [inline doc/manuals/admin/upgrading_crossing_1-3-0.md]: some => any

Comment (fgalan) [inline doc/manuals/admin/upgrading_crossing_1-3-0.md]: Fixed in a217a56c262be9f488922a5e939340e2b24e93a8

File: scripts/managedb/csub_merge_condvalues.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+from pymongo import MongoClient
+import json
+import sys
+from time import sleep
+
+CONDITIONS = "conditions"
+
+def update_ok(doc, n_conditions):
+
```
Comment (iariasleon) [inline scripts/managedb/csub_merge_condvalues.py]: it is recommendable to use the docstring in each method (good practice)

Comment (fgalan) [inline scripts/managedb/csub_merge_condvalues.py]: Fixed in 0dc11b3 (not only for csub_merge_condvalues.py, but also for in mdsvector2mdsobject.py).

Comment (iariasleon) [inline scripts/managedb/csub_merge_condvalues.py]: LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**